projects/demo/src/modules/components/input-tag/input-tag.component.ts

```diff
@@ -108,10 +108,9 @@ export class ExampleTuiInputTagComponent extends AbstractExampleTuiControl {
 
     inputHidden = false;
 
-    readonly disabledItemHandlerVariants: [
-        TuiBooleanHandler<string | TuiStringifiableItem<string>>,
-        TuiBooleanHandler<string>,
-    ] = [ALWAYS_FALSE_HANDLER, item => item[0] === 'T'];
+    readonly disabledItemHandlerVariants: TuiBooleanHandler<
+        string | TuiStringifiableItem<string>
+    >[] = [ALWAYS_FALSE_HANDLER, item => (item as string)[0] === 'T'];
```

Comment:
You can do `String(item)[0]` instead of typecasting.

Reply:
fixed